pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -9,6 +9,8 @@ python3Packages.buildPythonApplication rec {
     sha256 = "0qn5hyzvam3rimn7g3671s1igj7fbkwdnf5nc8jr4d5swy25mq61";
   };
 
+  patchPhase = "cat ${./depfixer_patchelf.py} >> mesonbuild/scripts/depfixer.py";
```

Comment:
This should probably be in `postPatch` in case we need to add patches in the future.